<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1592</width>
    <height>1577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xxx1.qrc">
    <normaloff>:/new/prefix1/icon</normaloff>:/new/prefix1/icon</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>选择串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtona">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>uaisa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonb">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>uaisb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonc">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>uaisc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtond">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>uaisd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>save pdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fastButton">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slowButton">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>slow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendtextEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slowButton2">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>170</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>slow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>170</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fastbutton2">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>170</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fastbutton3">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slowButton3">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>slow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slowButton4">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>slow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fastbutton4">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton4">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fastbutton5">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slowButton5">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>slow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton5">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton6">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>530</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slowButton6">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>530</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>slow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fastbutton6">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>530</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slowButton7">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>620</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>slow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton7">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>620</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fastbutton7">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>620</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slowButton8">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>710</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>slow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fastbutton8">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>710</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton8">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>710</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1171</width>
      <height>728</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1171</width>
      <height>728</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1171</width>
      <height>2435</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1171</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1171</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1171</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1171</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1171</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1171</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot4" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1171</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1171</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot5" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1171</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1171</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot6" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1171</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1171</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot7" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1171</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1171</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot8" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1171</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1171</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rescaleButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>rescale</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1592</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xxx1.qrc"/>
 </resources>
 <connections/>
</ui>
